### PR TITLE
sandboxes: Start Sandboxes

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/INDIGO-Initiative/database-app/issues/66

This is only available to transactions table in projects so far.
    
But it is a general purpose framework that could apply to any status field.

### CODE REVIEW

File: indigo/updatedata.py [hunk 1]
```diff
@@ -325,6 +336,68 @@ def filter_values(
...
 
 
+def get_sandbox_only_data(
+    data,
+    keys_with_own_status_subfield=[],
+    lists_with_items_with_own_status_subfield=[],
+):
+    sandbox_data = {}
+
+    # Some single values (or groups of single values) wight be removed, based on a status field.
```
Comment (Lathrisk) [inline indigo/updatedata.py:346]: _might_ be removed?

**Review by Lathrisk — APPROVED** [review]
a little typo but otherwise looks good to me

File: indigo/updatedata.py (hunk 1 above)
Comment (ghost) [reply indigo/updatedata.py:346]: I'll fix tomorrow, thanks

Comment (ghost) [reply indigo/updatedata.py:346]: Also these comments were copied and pasted and aren't actually correct. Which also means there is a typo further up the file too.

**Pull request merged**